surf.deckr.deckr.desktop

```diff
@@ -0,0 +1,13 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+
+Name=Deckr
+Comment=Companion app for the Deckr chrome plugin
+Categories=Utility;Amusement;
```

Comment:
is the second one a valid category?

Reply:
Why not?  Browsing internet is amusement.

Reply:
where do I find a list of categories.  I can't find anything on your site.

Reply:
Valid as in present here https://specifications.freedesktop.org/menu-spec/latest/apa.html

You can't put arbitrary category names.

Reply:
I don't see amusement there or in the next page.